src/main/kotlin/no/nav/helse/spade/Spade.kt

```diff
@@ -49,7 +49,8 @@ fun Application.spade() {
          verifier(jwkProvider, idProvider["issuer"].toString())
          realm = environment.config.propertyOrNull("ktor.application.id")?.getString() ?: "Application"
          validate { credentials ->
-            if (credentials.payload.getClaim("NAVident").asString() in authorizedUsers) {
+            if (credentials.payload.getClaim("NAVident").asString() in authorizedUsers &&
+               environment.config.property("clientId").getString() in credentials.payload.audience) {
```

Comment:
Muy bien 👍  Liten NB: i og med appen er satt opp med menneskevennlig "app id uri" i Azure så tror jeg faktisk det er to gyldige clientId'er og to gyldige audienser: både https://trygdeetaten.no/helsespade og UUIDen , og audience vil velges utifra hvilken variant klienten angir i score virker det som. Men veldig eksplisitt og fint med if (JEG er blant tilskuerne) i stedet for ev. "allowed_audiences" da :).   Burde vi ev. logget "feil audience" som feilårsak i tilfelle det skulle bli noe feilsøking rundt det en gang ?   (på den annen side vet jeg KTOR-folket virket veeeldig opptatt av å _ikke_ logge på info-nivå ved 401 i frykt for DOS-angrep, men speil/spade er vel ikke eksponert på internet uansett da)

Reply:
Logging av feilet validering gjør vi jo uansett, slenger på info om audience.